source/crud/tests/README.rst

```diff
@@ -61,6 +61,14 @@ Each YAML file has the following keys:
     and "sharded". If this field is omitted, the default is all topologies (i.e.
     ``["single", "replicaset", "sharded"]``).
 
+  - ``serverlessMode``: (optional) Whether or not the test should be run on
```

Comment:
May I humbly suggest `serverless: "allow|require|forbid"`? I feel it's less verbose and less confusing (`serverlessMode` makes it sound like it's about the configuration of the serverless proxy, not whether we're testing against it)

Reply:
Sounds good, updated.